src/index.d.ts

```diff
@@ -186,25 +186,50 @@ declare module '@bergfreunde/x-data-spreadsheet' {
      * @param colIndex
      * @param style
      * @param sheetIndex
+     * @param reRender
      */
      setCellStyle(
       rowIndex: number,
       colIndex: number,
       style: CellStyle,
-      sheetIndex: number
+      sheetIndex?: number,
+      reRender?: boolean
```

Comment:
```suggestion
      rerender?: boolean
```

Rerender ist nur ein Wort.